src/plugins/messenger/MessengerPreview/components/MessengerGenericTemplate/MessengerGenericTemplate.tsx

```diff
@@ -1,135 +1,155 @@
-import { IFBMGenericTemplatePayload, IFBMGenericTemplateElement } from '../../interfaces/GenericTemplatePayload.interface';
```

Comment:
please revert undesired styling changes. E.g.: This file now has tab stops of 2 spaces, we always use 4

Reply:
fixed